### DISCUSSION TITLE
Can I sandbox emerge on Gentoo Linux?

### DISCUSSION THREAD

**Discussion by crocket** [netblue30/firejail#5015] [Q&A] [answered]
It builds packages and installs them. It also creates its own network namespace for blocking internet access during build.

It requires root. I wonder whether firejail can deny access to certain directories effectively.

**Comment by kmk3** [comment, marked as answer]
> Is it possible to sandbox emerge, gentoo's package installer?

This looks like a duplicate of #4956.

**Reply by crocket** [reply]
Ouch.
